src/neptune_fetcher/read_only_run.py

```diff
@@ -113,3 +113,16 @@ def prefetch(self, paths: List[str]) -> None:
             paths: List of field paths to prefetch.
         """
         self._cache.prefetch(paths=paths)
+
+    def prefetch_series_values(self, paths: List[str], use_threads: bool = False) -> None:
+        """
+        Prefetches values of a list of series fields and stores them in local cache.
+
+        Args:
+            paths: List of field paths to prefetch.
+            use_threads: If True, fetching will be done concurrently.
+
+        To control the number of workers in the thread pool, set the environment variable
+         NEPTUNE_FETCHER_MAX_WORKERS. The default value is 10.
```

Comment:
```suggestion
         NEPTUNE_FETCHER_MAX_WORKERS environment variable. The default value is 10.
         
         Example:
         ```
         run.prefetch_series_values(["metrics/loss", "metrics/accuracy"])

         # No more calls to the API
         print(run["metrics/loss"].fetch_values())
         print(run["metrics/accuracy"].fetch_values())
         ```
```